### DISCUSSION TITLE
How to pass custom props to getCellProps()

### DISCUSSION THREAD

**Discussion by rossthedevigner** [TanStack/table#2687] [General]
As per the documentation: https://react-table.tanstack.com/docs/api/useTable#cell-properties

>getCellProps: Function(?props)
Required
This function is used to resolve any props needed for this cell.
You can use the getCellProps hook to extend its functionality.
Custom props may be passed. NOTE: Custom props will override built-in table props, so be careful!

Where and how does one extend this functionality?

Thanks!

**Comment by dkulakov1** [comment]
Hey, look on this example
https://react-table.tanstack.com/docs/examples/data-driven-classes-and-styles

**Reply by cjbland** [reply]
To those that have come here from Google now that v8 is out in the wild:

https://react-table-v7.tanstack.com/docs/examples/data-driven-classes-and-styles

**Reply by awreese** [reply]
@cjbland 404 on both these URLs. 🙁

**Reply by cjbland** [reply]
You can find it here now: https://react-table-v7-docs.netlify.app/docs/examples/data-driven-classes-and-styles
